### PR TITLE
feat(tabs): add actions section

### PR DESCRIPTION
**Pull request by dtsanevmw**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [ ] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?

1: Slot which is being projected inside the header so basically `<clr-tabs-actions>Whatever content you put here</clr-tabs-actions>`. 
2: Directive `clrTabsAction` which you put on the element you want to be interactable and part of the navigation of tabs ( arrows ).

- [ ] Bugfix
- [X] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: CDE-2391

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: .storybook/stories/tabs/tabs-actions.stories.ts [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    activeTab: 1,
+    title: 'Tab',
+    content: 'Tab Content',
+  },
+};
+
+const tabsTemplate: StoryFn = args => ({
+  template: `
+    <clr-tabs [clrLayout]="clrLayout">
+      <clr-tabs-actions position="right">
```
Comment (valentin-mladenov) [inline .storybook/stories/tabs/tabs-actions.stories.ts:54]: Viasual tests for `left` position is missiing.

File: projects/angular/src/layout/tabs/tabs-actions.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  host: {
+    '[class.tabs-actions]': 'true',
+  },
+})
+export class ClrTabsActions {
+  @Input()
+  @HostBinding('attr.position')
+  position: ClrTabsActionsPosition = 'right';
+
+  @ContentChildren(ClrTabAction, { read: ElementRef }) actions: QueryList<ElementRef>;
```
Comment (valentin-mladenov) [inline projects/angular/src/layout/tabs/tabs-actions.ts:30]: Is this used any where?

```suggestion
```

**Review by valentin-mladenov — APPROVED** [review]
CHeck my comments.

File: .storybook/stories/tabs/tabs-actions.stories.ts (hunk 1 above)
Comment (dtsanevmw) [reply .storybook/stories/tabs/tabs-actions.stories.ts]: Resolved in [2d0a306](https://github.com/vmware-clarity/ng-clarity/pull/1620/commits/2d0a306495298892d43ee3dd569062800ee44180)

File: projects/angular/src/layout/tabs/tabs-actions.ts (hunk 2 above)
Comment (dtsanevmw) [reply projects/angular/src/layout/tabs/tabs-actions.ts]: Resolved in [2d0a306](https://github.com/vmware-clarity/ng-clarity/pull/1620/commits/2d0a306495298892d43ee3dd569062800ee44180)

**Pull request merged**